aa_engine/aas_retrieve_inputs_part2.m

```diff
@@ -140,23 +146,37 @@
                                     oldpth=newpth;
                                 end;
                                 if ispc()
-                                    % On Windows, always use copyfile for now.
-                                    % TODO: Allow hardlinks on Windows? Can they even be created without admin rights?
                                     src_full = fullfile(streamfiles(depind).src, streamfiles(depind).fns{ind});
-                                    copyfile(src_full, streamfiles(depind).fns_dest_full{ind});
+                                    if (~streamfiles(depind).ismodified)
```

Comment:
Since the condition structure in lines 166-177 seems more elegant, I suggest adopting it here, as well.